swagger2.0/oic.sec.ael.swagger.json

```diff
@@ -0,0 +1,263 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "Security Event List",
+		"version": "2019-10-03",
+		"license": {
+			"name": "OCF Data Model License",
+			"url": "https://github.com/openconnectivityfoundation/core/blob/e28a9e0a92e17042ba3e83661e4c0fbce8bdc4ba/LICENSE.md",
+			"x-copyright": "Copyright 2019 Open Connectivity Foundation, Inc. All rights reserved."
+		},
+		"termsOfService": "https://openconnectivityfoundation.github.io/core/DISCLAIMER.md"
+	},
+	"schemes": ["http"],
+	"consumes": ["application/json"],
+	"produces": ["application/json"],
+	"paths": {
+		"/AelResURI": {
+			"get": {
+				"description": "This Resource contains the Auditable Events that have been logged on the Device.",
+				"parameters": [
+					{ 
+						"$ref": "#/parameters/interface" 
+					},
+					{
+						"ref": "#/parameters/events-filtered-by-di"
+					},
+					{
+						"ref": "#/parameters/events-filtered-by-category"
```

Comment:
We need to add a test for this if we are allowing filtering by "category"

Reply:
Removed filtering for now